src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -22,22 +22,23 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include=".\AlwaysPassTest.cs" />
-    <Content Include="dotnet-Microsoft.XmlSerializer.Generator.runtimeconfig.json">
```

Comment:
Keep this.

Reply:
Actually, if you wanted to avoid having this file you can just include $(XunitRuntimeConfig).  That's the same runtimeconfig used by the tests.